introduction.md

```diff
@@ -1 +1,4 @@
 I'm introducing myself and I like JavaScript path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?